Makefile

```diff
@@ -79,11 +80,12 @@ GEN_SRC = $(GEN_DIR)/both_mode_insns_table_body.adoc               \
           $(GEN_DIR)/Zcheri_mode_insns_table_body.adoc             \
           $(GEN_DIR)/Zcheri_purecap_insns_table_body.adoc
 
-# Docker command
-DOCKER = docker run --rm -v $(PWD):/build -w /build $(DOCKER_IMAGE)
+# Docker command. `-it` is necessary so that ctrl-c works.
+DOCKER = docker run -it --rm -v $(PWD):/build -w /build $(DOCKER_IMAGE)
```

Comment:
I think `-it` stands for "interactive", which is probably why the CI is failing. Could you please take a look?